core/defaultconfigs/dagent.py

```diff
@@ -1,14 +1,12 @@
-import os
+#!/usr/bin/env python3
```

Comment:
What sense does it make to have these python files with shebang?

Reply:
Its a script that is run, so my reasoning was the other way around. Why leave it out?
https://github.com/zokradonh/kopano-docker/blob/master/core/start-service.sh#L52

Reply:
Sure, but please either use a shebang or call it as a script with interpreter. Now its duplicated.